rbd-target-api.py

```diff
@@ -2621,6 +2621,8 @@ def target_ready(gateway_list):
                     "summary": ''}
 
     for gw in gateway_list:
+        if gw == "localhost"
```

Comment:
Need a ":" at the end of the line.

`if gw == "localhost":`

Reply:
One question just to make sure I understood the localhost check. Is it for cases like rbd-target-api:clientauth():

`    gateways.insert(0, 'localhost')
`
The check you are adding allows callers to just use localhost instead of doing 

gateways.insert(0, settings.config.api_host)

right?

Reply:
yes ,that's what I thought.